src/router.js

```diff
@@ -99,6 +101,22 @@ function createRouter() {
 					returnLabel: '',
 				},
 			},
+			{
+				path: '/dashboard/floor-1/:id',
+				name: 'floor-1',
+				component: PaintingDetails,
+				meta: {
+					title: 'MMT Dashboard',
+					returnPath: '',
+					returnLabel: '',
+				},
+			},
+			// {
+			// 	path: '/selection-overview',
+			// 	name: 'selection overview',
```

Comment:
Is this double or did a merge not pass by correctly

Reply:
no this is the example of the routes do we still need it !?